libvast/src/system/index.cpp

```diff
@@ -40,6 +40,7 @@
 #include "vast/json.hpp"
 #include "vast/load.hpp"
 #include "vast/logger.hpp"
+#include "vast/meta_index.hpp"
```

Comment:
We normally omit includes in cpp files if the target is already included in the associated header.